Analytics-CSharp/Segment/Analytics/Utilities/Storage.cs

```diff
@@ -335,16 +335,8 @@ public void UserInfoUpdate(IState state)
         {
             var userInfo = (UserInfo)state;
             WritePrefs(StorageConstants.AnonymousId, userInfo._anonymousId);
-
-            if (userInfo._userId != null)
-            {
-                WritePrefs(StorageConstants.UserId, userInfo._userId);
-            }
-
-            if (userInfo._traits != null)
-            {
-                WritePrefs(StorageConstants.Traits, JsonUtility.ToJson(userInfo._traits));
-            }
+            WritePrefs(StorageConstants.UserId, userInfo._userId);
+            WritePrefs(StorageConstants.Traits, JsonUtility.ToJson(userInfo._traits));
```

Comment:
`JsonUtility.ToJson` probably will complain about a `null` parameter. we should still check it